test/integration-test/src/main.rs

```diff
@@ -31,6 +31,7 @@ macro_rules! exec_test {
     ($test:expr) => {{
         info!("Running {}", $test.name);
         if let Err(e) = ($test.test_fn)() {
+            println!("Error in {}: {}", $test.name, e);
```

Comment:
This is just a bit better than what we have currently, but this still doesn't change the backtrace and doesn't show which particular line of code is causing the error.

I described my idea of fix in this issue: #421

Reply:
removed, will do in a separate PR